pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1035,12 +1035,17 @@ func (g *genericScheduler) selectNodesForPreemption(
 // This function is stable and does not change the order of received pods. So, if it
 // receives a sorted list, grouping will preserve the order of the input list.
 func filterPodsWithPDBViolation(pods []interface{}, pdbs []*policy.PodDisruptionBudget) (violatingPods, nonViolatingPods []*v1.Pod) {
+	pdbsAllowed := make([]int32, len(pdbs))
+	for i, pdb := range pdbs {
+		pdbsAllowed[i] = pdb.Status.PodDisruptionsAllowed
+	}
+
 	for _, obj := range pods {
 		pod := obj.(*v1.Pod)
 		pdbForPodIsViolated := false
 		// A pod with no labels will not match any PDB. So, no need to check.
 		if len(pod.Labels) != 0 {
```

Comment:
I suggest adding a ```continue``` to reduce indentation.

```
if len(pod.Labels) == 0 {
  continue
}
```

Reply:
if `len(pod.Labels) == 0`, we have to call not only continue, but also `nonViolatingPods = append(nonViolatingPods, pod)`.